### DISCUSSION TITLE
Can not see the last line of editor

### DISCUSSION THREAD

**Discussion by dlintw** [LazyVim/LazyVim#1501] [Q&A] [answered]
In the default lazyVIM's setting, I can not see the last line of  file, It is be overwrite by status line. Is this a feature or bug?  If this is a feature, How to disable it.

**Comment by dpetka2001** [comment]
`:h 'laststatus'`

**Comment by folke** [comment, marked as answer]
Sounds like a Neovim bug. Make sure to use a recent stable or nightly build. I don't have that issue

**Comment by dlintw** [comment]
I guess so, but I haven't install nightly build to try. Sometimes, it shows the last line, sometimes, it can't not work.
